### DISCUSSION TITLE
What is the latest CUDA version supported?

### DISCUSSION THREAD

**Discussion by MrMondragon** [bitsandbytes-foundation/bitsandbytes#1049] [Q&A] [answered]
What is the latest CUDA version supported?

**Comment by matthewdouglas** [comment, marked as answer]
The recent binary wheels on PyPI support only Linux x86-64. They include CUDA binaries as follows:

| bitsandbytes | CUDA |
| -----------|------|
| 0.42.0 | 11.0, 11.1, 11.4, 11.5, 11.7, 11.8, 12.0, 12.1, 12.2, 12.3 |
| 0.40.0 - 0.41.3 | 11.0, 11.1, 11.4, 11.5, 11.7, 11.8, 12.0, 12.1, 12.2 |
| 0.39.1 | 11.0, 11.1, 11.2, 11.3, 11.4, 11.5, 11.6, 11.7, 11.8, 12.0, 12.1 |
| 0.39.0 | 11.1, 11.2, 11.3, 11.5, 11.6, 11.7, 11.8, 12.0, 12.1 |
| 0.37.2 - 0.38.1 | 10.2, 11.0, 11.1, 11.2, 11.3, 11.4, 11.5, 11.6, 11.7, 11.8, 12.0, 12.1 |
| 0.37.0 - 0.37.1 | 10.2, 11.0, 11.1, 11.2, 11.3, 11.4, 11.5, 11.6, 11.7, 11.8, 12.0 |

As of today, the `main` branch successfully builds with the latest CUDA Toolkit 12.3.2.
